resources/validations/ui/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="FedRAMP OSCAL Validations" />
```

Comment:
NB: Perhaps we call it FedRAMP Validations for now? Our review teams will not have an interest in its OSCALiness.

```suggestion
    <meta name="description" content="FedRAMP Validations" />
```